### DISCUSSION TITLE
Why the search bar element not take all the space of the father element with the "width: 100%"?

### DISCUSSION THREAD

**Discussion by OriLiMu** [ScratchAddons/ScratchAddons#7551] [Help] [answered]
![image](https://github.com/ScratchAddons/ScratchAddons/assets/11295355/c782007b-2685-4252-be52-978b9a95aa34)
You can see it from the image as title said. Why the input-bar not take all the space of father element?

**Comment by mxmou** [comment, marked as answer]
It has a `max-width`.

**Comment by WorldLanguages** [comment]
@woshichuanqilz Please understand that this repository is not the correct place to ask general questions about web development, CSS, and extensions. Keep it to a minimum

**Reply by OriLiMu** [reply]
Is there a way to delete this discussion? 

I check  the post here. https://github.com/orgs/community/discussions/52581

But seems the "delete" button is not exist now

**Reply by WorldLanguages** [reply]
I don't see any reason to delete it

**Reply by DNin01** [reply]
You can close your discussions by using the button at the bottom of the page if your questions have been answered.

**Reply by OriLiMu** [reply]
> You can close your discussions by using the button at the bottom of the page if your questions have been answered.

There is no close button in the page. It is a **help** discussion I already mark the answer 
![image](https://github.com/ScratchAddons/ScratchAddons/assets/11295355/04b12998-ea3e-40cf-af54-15c19228828e)

**Reply by DNin01** [reply]
It's not showing up here because someone else already closed it.

**Reply by OriLiMu** [reply]
> It's not showing up here because someone else already closed it.

oh I know. Sorry. Already close other discussion which is done

**Comment by OriLiMu** [comment]
sorry，at first I dont know it is just  a css issue. I thought it was caused
by the some mechanism in addons

W_L ***@***.***> 于 2024年6月19日周三 23:36写道：
